### PR TITLE
feat(pci): add kubernetes to public-cloud app

### PR DESCRIPTION
**Pull request by marie-j**
# Add kubernetes 

cc \ @cbourgois @jleveugle @antleblanc

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/kubernetes/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import kubernetes from '@ovh-ux/manager-kubernetes';
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/kubernetes/index.js]:
```suggestion
import '@ovh-ux/manager-kubernetes';
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import routing from './routing';
+
+const moduleName = 'ovhManagerPciProjectKubernetes';
+
+angular
+  .module(moduleName, [
+    kubernetes,
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/kubernetes/index.js]:
```suggestion
    'ovhManagerKubernetes',
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/index.js (hunk 1 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/project/kubernetes/index.js]: Nop, it's in the same module

Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/project/kubernetes/index.js]: Due to the fact that you are importing a peerDependency, I encourage you to import the module as we already done across others modules.

Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/project/kubernetes/index.js]: You are right ;)

**Pull request merged**